src/Utilities/postprocess-python/README.md

```diff
@@ -0,0 +1,2 @@
+This directory should contain Python scripts 
```

Comment:
Until we have something, I would say lets remove this. An issue that this is needed is better than an empty file (in my opinion)